src/apps/reservation-list/modal/reservation-details/reservation-details.tsx

```diff
@@ -23,11 +25,18 @@ const ReservationDetails: FC<ReservationDetailsProps & MaterialProps> = ({
   openReservationDeleteModal
 }) => {
   const t = useText();
+  const config = useConfig();
   const { state, identifier, numberInQueue } = reservation;
   const { authors, pid, year, title, description, materialType } =
     material || {};
   const branches = useGetBranches("blacklistedPickupBranchesConfig");
+  const allowRemoveReadyReservation = config(
+    "reservationDetailAllowRemoveReadyReservationsConfig"
+  );
   const isDigital = !!reservation.identifier;
+  const allowUserRemoveReadyReservations =
+    (state === "readyForPickup" && strToBool(allowRemoveReadyReservation)) ||
```

Comment:
"readyForPickup" should be defined as an `object as const`or an `enum`.
Great with the `strToBool` function. I just think the name is too generic. Is it possible to give it a more descriptive name. What is the purpose of it?